.github/stale.yml

```diff
@@ -0,0 +1,19 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 180
```

Comment:
Once configured does this apply to all current issues immediately, i.e. will it basically close all issues that have not had any activity in 187 days?

Also, does this apply as soon as we merge to master or is there some additional configuration?

Reply:
> Once configured does this apply to all current issues immediately
I've done a few searches and can't find anything explicit, so I'm going to assume yes for safety. So there will be a mass closure hence I think it makes sense to enable this post-Mantid Plot migration

> Also, does this apply as soon as we merge to master or is there some additional configuration?
Provided that [Stale Bot](https://github.com/apps/stale) isn't enabled for the repo nothing will happen. At the point of adoption a repository admin can add this